YACS-RCOS/yacs.n
# 723 more detailed explore page with sorting options

**Issue**

closes #723 

**Test Procedure**

1. Go to Explore page
2. Use the side buttons to sort between Department and Alphabet
3. Switch between semesters to check all departments have their long name displayed (ex. Fall 2021 for STSO)

**Photos**
Before:
<img width="1401" alt="Screenshot 2023-07-14 at 2 27 39 PM" src="https://github.com/YACS-RCOS/yacs.n/assets/90737103/e20453b3-7131-4006-b33d-35c5a46136b9">

After:
<img width="1402" alt="Screenshot 2023-07-14 at 2 28 06 PM" src="https://github.com/YACS-RCOS/yacs.n/assets/90737103/801a0cdf-4c54-4097-a56b-e773ae13480f">
